apps/svelte.dev/content/blog/2026-01-01-whats-new-in-svelte-january-2026.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "What's new in Svelte: January 2026"
+description: 'Hydration with CSPs, automatic Cloudflare adapter setup, and AI works better now with Svelte!'
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+During the December holiday season, the Svelte team shared 24 days of Svelte knowledge, tips, and insights to unwrap in 2025's [Advent of Svelte](https://advent.sveltesociety.dev/2025). Learn more about under-utilized Svelte features through a series of fun and interactive videos!
```

Comment:
We should probably synchronize this announcement with a new or refined blog post based on <https://svelte.dev/blog/advent-of-svelte-is-back>. cc @paoloricciuti

Reply:
Agree, we don't need to synchronize/update the old blog post. Wouldn't link to it either since it's just adding one more jump to where you actually wanna go; the society overview page.